src/BTCPayServer.Lightning.CLightning/CLightningClient.cs

```diff
@@ -326,6 +332,35 @@ private async Task<PayResponse> PayAsync(string bolt11, PayInvoiceParams payPara
                         : PayResult.Error;
                 return new PayResponse(result, ex.Message);
             }
+            catch (Exception ex) when (ex is TaskCanceledException or OperationCanceledException)
```

Comment:
Change to `when (cts.IsCancelledRequested)`

Reply:
Done in [994aee3](https://github.com/btcpayserver/BTCPayServer.Lightning/pull/106/commits/994aee31d9bb4f4e204e3a9bf362ea1cef231221).